solutions/slackbot/api/_constants.js

```diff
@@ -0,0 +1,12 @@
+import { Redis } from '@upstash/redis'
```

Comment:
💡 (not a request for change, just an idea)

 I would consider moving those into a `lib` folder similar to the pattern in other examples. This would make the API folder easier to read as it would only contain routes.